ProcessLib/PhaseField/LocalAssemblerInterface.h

```diff
@@ -0,0 +1,100 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <vector>
+
+#include "NumLib/Extrapolation/ExtrapolatableElement.h"
+#include "ProcessLib/LocalAssemblerInterface.h"
+
+namespace ProcessLib
+{
+namespace PhaseField
+{
+
+struct PhaseFieldLocalAssemblerInterface
+    : public ProcessLib::LocalAssemblerInterface,
+      public NumLib::ExtrapolatableElement
+{
+    virtual std::vector<double> const& getIntPtSigmaXX(
```

Comment:
I suggest to also switch to vectorial output in this PR.

Reply:
There will be a separate PR switching mechanics processes to vectorial output.

Reply:
That's also OK.